app/charts/bar/chart-bar.tsx

```diff
@@ -53,7 +53,7 @@ const ChartBars = memo((props: ChartProps<BarConfig>) => {
         <StackedBarsChart {...props}>
           <ChartContainer>
             <ChartSvg>
-              <AxisHeightLinear />
+              <AxisWidthLinear />
               <AxisWidthBand />
```

Comment:
I didn't check the implementation yet, but shouldn't we have `AxisHeightBand`, instead of `AxisWidthBand`?